components/cms/Toolbar.tsx

```diff
@@ -27,15 +27,26 @@ export const Toolbar = styled(({ ...styleProps }) => {
           <CreateContentMenu />
         </Create>
         <Github>
-          {git.all().length > 0 && git.all().map((git: any) => git.component())}
+          {git.all().length > 0 &&
+            git
+              .all()
+              .map((git: any, i) => <git.component key={`${i}-${git.name}`} />)}
```

Comment:
On second look I would also do this:

```suggestion
              .map((git: any, i) => <git.component key={git.name} />)}
```

Using the array index in the `key` is [generally not advised](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318)

Also, `name` is unique across plugins so you'll never have two plugins of the same type with the same.

Reply:
okay, I wasn't sure if there would be multiple instances of a single plugin